paint-prototype/Assets/Fork/Lauren/PaintTank.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class PaintTank : MonoBehaviour
+{
+
+    public Slider slider;
+    public Gradient gradient;
+    public Image fill;
+
+    public void SetMaxHealth(int health)
+    {
+        slider.maxValue = health;
+        slider.value = health;
+
+        fill.color = gradient.Evaluate(1.0f);
+    }
+
+    public void SetHealth(int health)
+    {
+        slider.value = health;
+        fill.color = gradient.Evaluate(slider.normalizedValue);
+
+    }
```

Comment:
Add a removePaint method instead? instead of setting things it just removes from a set amount
Means we don't have to get the current amount of paint every time we want to add or remove.
Should also add an addPaint method for refilling